source/index.md

```diff
@@ -479,35 +506,119 @@ POST https://gateway.clearhaus.com/authorizations
-  <dt>card[number]</dt>
+
+  <!-- deprecated -->
+  <dt><strike>threed_secure[pares]</strike></dt>
+  <dd>
+    Deprecated! Please use <code>card[pares]</code>. <br />
+    [:base64:] <br />
+    See more information at <a target="_blank" href="http://docs.3dsecure.io">3Dsecure.io</a>.
+  </dd>
+  <dt><strike>card[number]</strike></dt>
+  <dd>
+    Deprecated! Please use <code>card[pan]</code>. <br />
+    [0-9]{12,19} <br />
+    Primary account number of card to charge.
+  </dd>
+</dl>
+
+##### Method: `card`
+
+<dl class="dl-horizontal">
+  <dt>card[pan]</dt>
   <dd>[0-9]{12,19} <br /> Primary account number of card to charge.</dd>
   <dt>card[expire_month]</dt>
   <dd>[0-9]{2} <br /> Expiry month of card to charge.</dd>
   <dt>card[expire_year]</dt>
   <dd>20[0-9]{2} <br /> Expiry year of card to charge.</dd>
   <dt>card[csc]</dt>
-  <dd>[0-9]{3} <br /> Card Security Code.</dd>
+  <dd>
+    [0-9]{3} <br />
+    <i>Optional when transaction is signed and partner is trusted.</i> <br />
+    Card Security Code.
+  </dd>
+  <dt>card[pares]</dt>
+  <dd>[:base64:] <br /> <i>Optional</i> <br /> See more information at <a target="_blank" href="http://docs.3dsecure.io">3Dsecure.io</a></dd>
+</dl>
+
+<p class="alert alert-info">
+  <b>Notice:</b> An approved authorization that includes
+  <code>card[pares]</code> is 3-D
+  Secured and is strongly authenticated.
+  <br />
+  <b>Notice:</b> An authorization that includes <code>card[pares]</code> and/or
+  <code>card[csc]</code> cannot be a subsequent recurring authorization.
+</p>
+
+##### Method: `applepay`
+
+Apple Pay requires the payment details (PAN, expiry, etc.) of the payment token
+to be decrypted by a symmetric key.
+Your systems must derive this symmetric key using the payment token's ephemeral
+public key, the merchant's private key and the merchant ID. For this, we refer
+to [our reference implementation](https://github.com/clearhaus/pedicel) written
+in Ruby; see [Apple's documentation for the <code>PaymentToken</code>
+object][ApplePay-PaymentToken] for more information.
+
+<dl class="dl-horizontal">
+  <dt>applepay[payment_token]</dt>
+  <dd>[:json:] <br /> Full, raw <code>PKPaymentToken</code> object, UTF-8 encoded serialization of a JSON dictionary.</dd>
+  <dt>applepay[symmetric_key]</dt>
+  <dd>[:hex:] <br /> Symmetric AES key (unique per transaction) that can
+    decrypt <code>data</code> from the <code>PKPaymentToken</code>.</dd>
 </dl>
 
 <p class="alert alert-info">
-<b>Notice:</b> The card dictionary is not required when making an authorization
-based on a card resource. However, <code>card[csc]</code> should be populated
-when CSC is available.
+  <b>Notice:</b> Signing is required to use the <code>applepay</code> payment
+  method.
+  <br />
+  <b>Notice:</b> An authorization made with <code>applepay</code> is
+  strongly authenticated.
```

Comment:
Perhaps we should explain "strongly authenticated"? This is the first occurrence of the term.

Reply:
We could add e.g. "(SCA in PSD2)"...

@kse-clearhaus WDYT?